src/i18n/fr/tester.js

```diff
@@ -0,0 +1,35 @@
+export default {
+  page: {
+    label: 'Testeur De Clavier'
+  },
+  keycodeStatus: {
+    label: 'Touches Détectées'
+  },
+  back: {
+    label: 'Retour',
+    title: 'Retour au configurateur'
+  },
+  reset: {
+    label: 'Réinitialiser',
+    title: 'Réinitialise les touches détectées'
+  },
+  letters: {
+    keycode: {
+      label: 'code de la touche'
+    },
+    code: {
+      label: 'code'
+    },
+    key: {
+      label: 'touche'
+    }
+  },
+  docs: {
+    paragraph:
+      'Note: La touche et le code détecté.es peuvent varier selon votre localisation. Docs'
```

Comment:
Not sure about the inclusive writing here, looks like a typo. Also the good dot character is this one `·` no the classic `.`.